config.js

```diff
@@ -33,9 +33,6 @@ var config = {
     // Websocket URL
     // websocket: 'wss://jitsi-meet.example.com/xmpp-websocket',
 
-    // The name of client node advertised in XEP-0115 'c' stanza
-    clientNode: 'http://jitsi.org/jitsimeet',
```

Comment:
Please don't delete this.

Reply:
Damn! Sorry I don't remember how I just deleted this :(